src/shims/ApiCompatBaseline.netcoreapp.netstandard20Only.txt

```diff
@@ -0,0 +1,2 @@
+DEFAULT_APPNAME Error: 0 : Unable to resolve assembly 'Assembly(Name=System.Private.CoreLib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e)' referenced by the implementation assembly 'Microsoft.Cci.DummyModule'.
```

Comment:
See commit message in https://github.com/dotnet/standard/commit/df833b04140b91f6d6bf264de8e42840f528f1e0. This is a result of CCI not being able to determine the core assembly.